### PR TITLE
Publications Update

### PR DESCRIPTION
**Pull request by baylorfk**

### CODE REVIEW

File: doc/bibliography/published.bib [hunk 1]
```diff
@@ -7,6 +7,113 @@
...
+	Doi = {10.5065/g8w0-fy32},
+	Journal = {CLIVAR Exchanges/US CLIVAR Variations},
+	Month = {February},
+	Number = {1},
+	Pages = {37--41},
+	Title = {GEOMETRIC: Geometry and energetics of ocean mesoscale eddies and their representation in climate models},
+	Url = {https://doi.org/10.5065/g8w0-fy32},
+	Volume = {18},
+	Year = {2020},
+	Bdsk-Url-1 = {https://doi.org/10.5065/g8w0-fy32}}
```
Comment (rabernat) [inline doc/bibliography/published.bib]: This looks like a duplicate, which may be why the build test [failed](https://travis-ci.com/ocean-eddy-cpt/ocean-eddy-cpt.github.io/builds/152655200).

File: doc/bibliography/published.bib [hunk 2]
```diff
@@ -7,6 +7,113 @@
...
+
+
 @article{TarshishEtAl2018,
 	Author = {Tarshish, Nathaniel and Abernathey, Ryan and Zhang, Ci and Dufour, Carolina O and Frenger, Ivy and Griffies, Stephen M},
 	Date-Added = {2018-09-22 14:16:54 +0000},
```
Comment (rabernat) [inline doc/bibliography/published.bib]: Now that we have some real publications in here, we might as well remove these dummy publications I put in as placeholders. Would you mind doing that as part of your PR?

File: doc/bibliography/published.bib (hunk 1 above)
Comment (baylorfk) [reply doc/bibliography/published.bib]: OK.  Removed the duplicate and the dummy papers

**Comment by rabernat** [comment]
That little green check mark means that the website built with no errors. Thanks!

**Comment by baylorfk** [comment]
Thanks--I haven't used the online editor function much.  I tend to clone to
my own workspace, but I think it's easier when I can't approve pushes
myself to work online.  Sorry about the buggy start.
  Cheers,
    -Baylor

Baylor Fox-Kemper
Brown University, Associate Professor
baylor@brown.edu, fox-kemper.com
401-863-3979

**Pull request merged**